build/scripts/tests.sh

```diff
@@ -20,10 +20,15 @@ xunit_console_version="$(get_package_version dotnet-xunit)"
 
 if [[ "${runtime}" == "dotnet" ]]; then
     target_framework=netcoreapp2.0
+    dir_list="${unittest_dir}"/*/netcoreapp2.0/*.UnitTests.dll
```

Comment:
I would probably prefer:

`dir_list=( "${unittest_dir}"/*/netcoreapp2.0/*.UnitTests.dll )`

and `for file_name in "${dir_list[@]}"` (and removal of `dir_list="${unittest_dll_list[@]}"`).

The way you have it now is susceptible to spaces in paths, I think - or at least really confusing (I don't think variable assignments expand globs, so the expansion is happening at for-loop-time - while being a statically known list for mono. Using an array (my preference above) makes the expansion happen at declaration-time)

Reply:
Actually, how does this work in the first place?? My testing:

```
khyperia@carbon ~ i % foo=*
khyperia@carbon ~ i % echo $foo
*
khyperia@carbon ~ i % for x in $foo; do; echo $x; done
*
khyperia@carbon ~ i % bar=( * )
khyperia@carbon ~ i % echo $bar
file1 file2 file3
khyperia@carbon ~ i % for x in "${bar[@]}"; do; echo $x; done
file1
file2
file3
khyperia@carbon ~ i %
```

Reply:
The variable should also probably be called `file_list` as well